src/client/editor/project.js

```diff
@@ -149,16 +168,16 @@ let projectTab = function() {
 			document.getElementById("paramAuthorName").addEventListener("input", function() { projectTab.setAuthorName( this.value ); }, false);
 
 			// Change default enter period
-			document.getElementById("paramEnterPeriod").addEventListener("change", function() { projectTab.setEnterPeriod( parseInt(this.value) ); }, false);
-			document.getElementById("paramEnterPeriod").addEventListener("input", function() { projectTab.setEnterPeriod( parseInt(this.value) ); }, false);
+			_elements.paramEnterPeriod.addEventListener("change", function() { projectTab.setEnterPeriod( parseInt(this.value) ); }, false);
```

Comment:
Although not really related to this PR, why  
`...addEventListener("change", function() { projectTab.setEnterPeriod( parseInt(this.value) ); }, false);`
instead of
`...addEventListener("change", projectTab.setEnterPeriod, false);` and handle the `parseInt()` inside the function? I think that would be a little cleaner.